napalm_logs/config/junos/ESWD_BPDU_BLOCK_ERROR_DISABLED.yml

```diff
@@ -0,0 +1,15 @@
+# This error tag corresponds to syslog messages notifying that the configured
```

Comment:
In general, the filename of the profile is the value of the `error`, so it's easier to check where is a feature defined; however, that's not mandatory.